### PR TITLE
Prevent two things from bumping up line height

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@jmbowman who will review styling changes?

**Comment by jmbowman** [comment]
Good question; maybe somebody from the FedX team?  They expressed interest in updating the theme to use the pattern library, see https://openedx.atlassian.net/browse/FEDX-253 .  Might be a good chance for some of them to start getting familiar with the theme's current CSS to get ready for that.

**Comment by nedbat** [comment]
I doubt they want to muck about with the mess they are going to replace :)

**Comment by nedbat** [comment]
I added another tweak to prevent another line-height bump.

**Comment by nedbat** [comment]
@edx/fedx-team anyone interested in reviewing a poor backend guy's CSS tweaks?

File: edx_theme/static/css/theme.css [hunk 1]
```diff
@@ -2606,6 +2606,7 @@ code, .rst-content tt {
 	background: #fff;
 	border: solid 1px #e1e4e5;
 	font-size: 75%;
+	line-height: 0px;
```
Comment (bjacobel) [inline edx_theme/static/css/theme.css]: Nit: zero values are unitless in CSS

Comment (nedbat) [inline edx_theme/static/css/theme.css]: Nice, amended.

**Comment by bjacobel** [comment]
👍

**Pull request merged**